<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmConfigDb</class>
 <widget class="QWidget" name="frmConfigDb">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>134</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labDbType">
     <property name="text">
      <string>主机类型</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="cboxDbType"/>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="labDbName">
     <property name="text">
      <string>数据库名</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="txtDbName"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labHostName">
     <property name="text">
      <string>主机地址</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="txtHostName"/>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="labHostPort">
     <property name="text">
      <string>通信端口</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLineEdit" name="txtHostPort"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labUserName">
     <property name="text">
      <string>用户名称</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="txtUserName"/>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="labUserPwd">
     <property name="text">
      <string>用户密码</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLineEdit" name="txtUserPwd">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QPushButton" name="btnConnect">
     <property name="text">
      <string>连接测试</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QPushButton" name="btnInit">
     <property name="text">
      <string>初始化数据</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
